app/templates/play/ladder_home.jade

```diff
@@ -23,5 +23,8 @@ block content
               if playCount
                 span.spl.spr - #{playCount.sessions}
                 span(data-i18n="play.players") players
+                if view.levelStatusMap[level.id]
+                  | &nbsp;-&nbsp;
```

Comment:
Use `.spr` and `.spl` instead of `&nbsp;`.

https://github.com/codecombat/codecombat/wiki/Add-space-between-elements